docs/get_started/build_from_source.md

```diff
@@ -337,38 +333,36 @@ other python libraries for MXNet.
 
 The Python package can be installed by one of the following three ways:
 
-1. Setup the environment variable `PYTHONPATH=/path/to/mxnet/python`. For example, assume `mxnet` is
-   placed on the home directory, then add the following line in the `rc` file
+1. Setup the environment variable `PYTHONPATH=/path/to/mxnet/python`. For example, assuming `mxnet` is
+   cloned in the home directory, then we need to add `~/mxnet/python` to `PYTHONPATH` in the `rc` file
    (e.g. `~/.bashrc`):
```

Comment:
since this location update to $PYTHONPATH is temporary and used only during installation, you don't need to .bashrc